observability/WFLY-19846-promote-mp-tel-2-to-default.adoc

```diff
@@ -0,0 +1,115 @@
+---
+categories:
+- microprofile
+- observability
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/619
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  -
+promotes: https://issues.redhat.com/browse/WFLY-19590
+promoted-by: https://github.com/wildfly/wildfly-proposals/issues/619
+---
+= [Default] Promote MicroProfile Telemetry 2.0 to WildFly Standard
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile Telemetry 2.0 shipped with WildFly Preview 34. To meet WildFly and product roadmap goals, this feature needs to be migrated to the WildFly feature pack, and its stability level promoted to `DEFAULT`.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Telemetry 2.0 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-19846[]
+
+=== Related Issues
```

Comment:
I noticed that telemetry PR adds grpc dependency [here](https://github.com/wildfly/wildfly/pull/18305/files#diff-0309302dfed110d75a04cb208108f436ddebfb77aed51fc8bca7d4155c7fdb4aR31). Is this telemetry RFE blocked by grpc RFE? (https://issues.redhat.com/browse/EAP7-1819 / https://issues.redhat.com/browse/WFLY-15495 ) If yes, I believe that it should be stressed out here.

Reply:
@marekkopecky No, it doesn't, this is just an internal dependency.

Reply:
Thank you for your feedback! I'm not sure whether I understand. Is this dependency in module.xml not used in module.xml in built WF?

Reply:
@marekkopecky That is an internal module dependency that is being added independently of the proposed gRPC subsystem. At some point, we'll have to work out the overlapping module definitions between the two, but there is no relationship at the moment.